haskell-src/Concordium/ID/Types.hs

```diff
@@ -1058,7 +1058,7 @@ values (NormalACWP cdi) = NormalAC (cdiValues cdi) <$> proofCommitments (cdiProo
 class HasCredentialType a where
   credentialType :: a -> CredentialType
 
-instance HasCredentialType AccountCredential where
```

Comment:
I don't see why this change is relevant for the transaction logging, but I see no problem in it

Reply:
It is not relevant.

But I needed it when I was updating the dependency in concordium-client (so I could update it in the wallet-proxy).

It is a leftover from the "improve startup speed" PR and since it's so trivial I did not want to make a separate PR with it.